prow/istio-pilot-e2e-envoyv2-v1alpha3-k8s-latest.sh

```diff
@@ -27,7 +27,7 @@ set -u
 # Print commands
```

Comment:
Why change the prefix of filename?

Reply:
I renamed istio-pilot-e2e-envoyv2-v1alpha3-k8s-1.10.sh to istio-pilot-e2e-envoyv2-v1alpha3-k8s-latest.sh and also deleted the unused e2e-pilot-no_auth-k8s-1.10.sh. Somehow git thinks I renamed e2e-pilot-no_auth-k8s-1.10.sh to istio-pilot-e2e-envoyv2-v1alpha3-k8s-latest.sh.